ansible/my_ansible_playbook.yml

```diff
@@ -0,0 +1,68 @@
+- name: Update Managed Hosts
+  hosts: managedhosts
+  remote_user: ec2-user
+
+  vars_prompt:
+
+  - name: trello_api_key
+    prompt: Enter your Trello API key..
+  
+  - name: trello_api_token
+    prompt: Enter your Trello API Token..
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: yes
```

Comment:
Not a suggestion as such, but worth noting that we can combine these two easily:
```yml
  - name: Install Git & Python3
    ansible.builtin.yum:
      name: 
        - git
        - python3
      state: latest
    become: yes
```